reference/6/Microsoft.PowerShell.Utility/ConvertFrom-Csv.md

```diff
@@ -12,130 +12,148 @@ title: ConvertFrom-Csv
 # ConvertFrom-Csv
 
 ## SYNOPSIS
-Converts object properties in comma-separated value (CSV) format into CSV versions of the original objects.
+Converts object properties in comma-separated value (CSV) format into CSV versions of the original
+objects.
 
 ## SYNTAX
 
 ### Delimiter (Default)
+
 ```
-ConvertFrom-Csv [[-Delimiter] <Char>] [-InputObject] <PSObject[]> [-Header <String[]>] [<CommonParameters>]
+ConvertFrom-Csv [-InputObject] <PSObject[]> [[-Delimiter] <Char>] [-Header <String[]>] [<CommonParameters>]
 ```
 
 ### UseCulture
+
 ```
-ConvertFrom-Csv [-UseCulture] [-InputObject] <PSObject[]> [-Header <String[]>] [<CommonParameters>]
+ConvertFrom-Csv [-InputObject] <PSObject[]> [-UseCulture] [-Header <String[]>] [<CommonParameters>]
```

Comment:
Correct syntax:
ConvertFrom-Csv [-InputObject] <psobject[]> -UseCulture [-Header <string[]>] [<CommonParameters>]

Reply:
Corrected the syntax.